parm/atm/obs/config/satwind_leogeo_multi.yaml.j2

```diff
@@ -0,0 +1,260 @@
+- obs space:
```

Comment:
@danholdaway this now belongs in JCB-GDAS, yes?

Reply:
That is correct. Sorry @BrettHoover-NOAA! You would add these to https://github.com/NOAA-EMC/jcb-gdas

Reply:
@CoryMartin-NOAA @danholdaway I am going to convert this to draft while I make the changes, I want to make sure this runs on the latest GDASApp configuration, the one I tested this on is clearly aged out

Reply:
YAML moved to https://github.com/NOAA-EMC/jcb-gdas/pull/4